source/compatibility.tex

```diff
@@ -2104,7 +2104,7 @@
 promising to return a value of a given type, and then not returning
 such a value, has always been recognized to be a questionable
 practice, tolerated only because very-old C had no distinction between
-void  functions and  int  functions.
+\tcode{void} functions and \tcode{int} functions.
```

Comment:
Let's fix this properly while we're here: "functions with \tcode{void} and \tcode{int} return types"